featurepack/test/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.hibernate.ogm</groupId>
+        <artifactId>hibernate-ogm-featurepack</artifactId>
+        <version>5.4.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>hibernate-ogm-featurepack-test</artifactId>
+    <packaging>pom</packaging>
+
+    <name>Hibernate OGM WildFly Feature Pack - Demonstration module</name>
+    <description>This is module shows how to provision a WildFly distribution with Hibernate OGM</description>
```

Comment:
Sure is not a "test" right. Maybe we can call this module "example" ?
If we move it into integration testing, we could use it as a basis for integration testing. 
At that point we could call it "wildfly-provisioned" or something.

Reply:
BUt doi you need this in the integration testing? Aren't you already creating the wildfly with all the dialects to run the existing tests?

I'm fine with renaming it `example` and keep it in the feature pack module. An alternative could be to add it in the `documentation/examples` with the gettingstarted one. If the purpose is to show the users how one can use feature packs, maybe this last one is the best approach

Reply:
No we don't need it.

Reply:
In documentation/examples could even better. Maybe we can call it "*-wildfly-quickstart" ?

Reply:
In the -wildfly-quickstart documentation can I add sample code and test as in gettingstarted?

Reply:
I don't know, I guess you can. Maybe keep all this part in a separate commit so that you can work on it and we can merge it without affecting the rest